data-collection/src/test/java/integration/toyprojects/R4ToyProjectTest.java

```diff
@@ -22,7 +22,7 @@ protected String getRepo() {
 	@Test
 	public void refactorings() {
 		List<RefactoringCommit> refactoringCommitList = getRefactoringCommits();
-		Assert.assertEquals(6, refactoringCommitList.size());
+		Assert.assertEquals(7, refactoringCommitList.size());
```

Comment:
Why do we have one more here? Did we change the repo? Or the code now captures one more refactoring because it's better?

Reply:
We capture another refactoring, we missed before.

Reply:
![image](https://user-images.githubusercontent.com/29139613/80352618-6af13d00-8874-11ea-875d-e22604ab2219.png)